apps/api/src/app/controllers/__tests__/fetchManyViewModels.e2e.spec.ts

```diff
@@ -66,6 +62,14 @@ describe('When fetching multiple entities', () => {
             const endpointUnderTest = `/${buildViewModelPathForEntityType(entityType)}`;
 
             describe(`GET ${endpointUnderTest}`, () => {
+                beforeEach(async () => {
+                    await testRepositoryProvider.testSetup();
```

Comment:
This is similar to python's setUp and tearDown?

Reply:
Yes. Most testing frameworks for any language have a similar concept.